nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/Analyzer.java

```diff
@@ -139,6 +139,10 @@ public void calculateThreshold() {
         int size = dataset.getSizeTotal(0);
         Vec vec = new Vec(size);
 
+        if (dataset.getComplex(0)) {
+            vec.resize(size / 2, true);
+        }
+
```

Comment:
For the complex datasets the vector ended up being the total size (ex. 16384) long instead of the complex size (ex. 8192) long which resulted in the second half of the vector being full of zeros. This resulted in one of the JCAMP datasets (INEPT_nd) having an exception when trying to integrate. It also meant that for complex datasets, the sDev value on line 160 will always be 0.  https://github.com/nanalysis/nmrfx/blob/36175bede8eaeb668aef62456697c557bdc545f1/nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/Analyzer.java#L160